src/helpers/get-offers-location.test.ts

```diff
@@ -0,0 +1,26 @@
+import { makeFakeOffers } from '../utils/mocks/offers';
+import { getOffersLocation } from './get-offers-location';
+
+describe('Function: getOffersLocation', () => {
+  const mockOffers = makeFakeOffers();
+
+  it('Return an array with {id, location} objects for map is correct', () => {
+    const offersLocation = mockOffers.map((offer) => ({
+      id: offer.id,
+      location: offer.location,
+    }));
+    const result = getOffersLocation(mockOffers);
+
+    expect(result).toEqual(offersLocation);
```

Comment:
```suggestion
    expect(result).toHaveProperty('id');
     expect(result).toHaveProperty('location');
```

https://jestjs.io/docs/expect#tohavepropertykeypath-value